secp256k1-sys/src/lib.rs

```diff
@@ -680,6 +680,15 @@ impl<T> CPtr for [T] {
 
 #[cfg(fuzzing)]
 mod fuzz_dummy {
+    /// Serialization logic:
+    /// If pk is created from sk keypair:
+    ///     - It is serialized with prefix 0x02: sk || [0xaa;32]
+    /// If pk is created from from slice:
+    ///     - 0x02||pk_bytes -> pk_bytes||[0xaa;32]
+    ///     - 0x03||pk_bytes -> pk_bytes||[0xbb;32]
+    ///     - 0x04||pk_bytes -> pk_bytes
```

Comment:
These types of patterns are *exactly* the kind of things fuzzers are good at finding. Why not just store an extra byte in memory?

Reply:
Is exceeding the 64 bytes and using space beyond the 64 bytes a good idea? 
Because we need all of 64 bytes to support full roundtrip of uncompressed keys.

Reply:
I had another related question, why do you need to serialize/parse functions for fuzzing anyway? Can we just not use regular logic for it. 

For forging signatures, the keys are anyways created by `secp256k1_ec_pubkey_create`

Reply:
No, we cannot switch to requiring public keys be valid according to standard rules. Not only does it break the RL fuzzers (see #271) but its also really slow, which can make fuzzers untenable.